Makefile

```diff
@@ -14,8 +14,8 @@ SYSTEM_DEPENDENCIES= \
 	virtualenv \
 	wget
 OS=$(shell lsb_release -si)
-PYTHON_MAJOR_VERSION=3
-PYTHON_MINOR_VERSION=6
+PYTHON_MAJOR_VERSION=$(shell python -c 'import sys;print(sys.version_info[0])')
+PYTHON_MINOR_VERSION=$(shell python -c 'import sys;print(sys.version_info[1])')
```

Comment:
Agr I'm sorry I thought I posted that comment already, but not :confused: 
https://github.com/kivy-garden/garden.zbarcam/pull/35/files#r261878306
Basically I explain that this is hardcoded to Python3 on purpose, and making it that dynamic would not work for system running e.g. Python2 by default.
So what I suggest is to keep it hardcoded with the option to override it. Good think is overriding variable is a built-in Makefile feature.
```sh
make PYTHON_MAJOR_VERSION=3 PYTHON_MINOR_VERSION=7
```
Let me know what you think

Reply:
Ok, let's do that